R/read_servers.R

```diff
@@ -34,29 +34,38 @@
 #' @examples
 #' \dontrun{
 #' data <- sql_server_read_data(
-#'   user          = "rfamro",
+#'   base_url      = "mysql-rfam-public.ebi.ac.uk",
```

Comment:
`base_url` here seems to have a slightly different definition than in other places. Did you double check it works by passing `"https://mysql-rfam-public.ebi.ac.uk"`. If not, you need to strip "https://" internally **OR** add it to the others for consistency.

I'm leaving it up to you to choose the best solution but it needs to be consistent.

Reply:
Adding `https://` to `mysql-rfam-public.ebi.ac.uk`  does not work.

Usually, users will provide the URL together with `https://`.
I don't think that it would be a good idea to explicitly mentione in the documentation that **URL should be provided without https:://** because our test server does not need it.
Also, having to handle `https://` internally means that it will be pasted to url supplied by the user and this has to be hard-coded in the functions. When implemented that way, examples that use this test server will no longer work as it does not require `https://`. I will have to allow for an exception in the functions to let those examples run fine.

I would suggest to let the user provide the URL as it works for the server to prevent any unexpected behaviour later on.

Reply:
Let's open an issue to continue this conversation because I can see how a potential user would be confused about the fact they sometimes have to pass `"example.com"` and sometimes `"https://example.com/"`, even though I understand the underlying technical reason.